server/app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const path = require('path');
+const cookieParser = require('cookie-parser');
+const logger = require('morgan');
+
+const controllers = require('./controllers');
+
+const app = express();
+
+app.use(logger('dev'));
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+app.use(cookieParser());
+app.use(express.static(path.join(__dirname, 'client', 'build')));
+
+app.use(controllers);
+app.get('/', (req, res) => {
+  res.sendFile(path.join(__dirname, '..', 'client', 'build', 'index.html'));
+});
+
```

Comment:
Would be good to add error handling for 404 / 500

Reply:
All done!